com.brosix.Brosix.metainfo.xml

```diff
@@ -18,6 +18,7 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="4.7.2 Build 220525.13731" date="2022-05-25"/>
```

Comment:
if you can also fix the app-id in the appstream file. `.desktop` is totally deprecated.

Reply:
Done, thanks!